OPAL/tac/src/main/scala/org/opalj/tac/fpcf/analyses/cg/AndroidICCAnalysis.scala

```diff
@@ -307,155 +323,165 @@ class AndroidICCAnalysis(val project: SomeProject) extends FPCFAnalysis {
     /**
      * Searches all Intents and callbacks in the project.
      */
-    def searchIntentsAndCallbacks(): Unit = {
-        val source = Source.fromInputStream(this.getClass.getResourceAsStream("/org/opalj/tac/fpcf/analyses/cg/AndroidCallbackList.txt"))
-        val callbackList = source.getLines.toList
-        source.close
+    def searchIntentsAndCallbacks(): ProperPropertyComputationResult = {
         val dec = project.get(DeclaredMethodsKey)
-        val leftOverMethods = mutable.Queue[Method]()
+        val partialResults = ListBuffer.empty[ProperPropertyComputationResult]
         project.allMethodsWithBody.filter(dec(_) != null).foreach(m ⇒ {
             val tacaiEP = propertyStore(m, TACAI.key)
             if (tacaiEP.hasUBP && tacaiEP.ub.tac.isDefined) {
-                processMethod(m, callbackList)
-            } else leftOverMethods += m
-        })
-        while (leftOverMethods.size > 1) {
-            val m = leftOverMethods.dequeue()
-            val tacaiEP = propertyStore(m, TACAI.key)
-            if (tacaiEP.hasUBP && tacaiEP.ub.tac.isDefined) {
-                processMethod(m, callbackList)
+                processMethod(m)
             } else {
                 leftOverMethods += m
+                partialResults += InterimPartialResult(Nil, Set(tacaiEP), continuationForTAC(m))
             }
+        })
+        if (leftOverMethods.isEmpty) {
+            startSecondPhase(project)
+        } else Results(partialResults)
+    }
+
+    private[this] def continuationForTAC(m: Method)(eps: SomeEPS): PropertyComputationResult = {
+        eps match {
+            case UBP(tac: TACAI) if tac.tac.isDefined ⇒
+                processMethod(m)
+                leftOverMethods -= m
+                if (leftOverMethods.isEmpty && notStarted) {
+                    notStarted = false
+                    startSecondPhase(project)
+                } else NoResult
+            case _ ⇒
+                InterimPartialResult(
+                    Nil,
+                    Set(eps),
+                    continuationForTAC(m)
+                )
         }
     }
 
-    private def processMethod(m: Method, callbackList: List[String]) = {
+    private def processMethod(m: Method): PropertyComputationResult = {
```

Comment:
Why does this return a `PropertyComputationResult` when that is always `NoResult` and seems to be unused?